workflow-bot-app/src/test/java/com/symphony/bdk/workflow/BranchingIntegrationTest.java

```diff
@@ -67,6 +75,24 @@ void branching(String workflowFile, List<String> activities) throws IOException,
     assertThat(workflow).isExecutedWithProcessAndActivities(lastProcess(), activities);
   }
 
+  @Test
+  void branching_intermediate() throws IOException, ProcessingException, InterruptedException {
```

Comment:
isn't this test similar to the one done here? 
test: https://github.com/finos/symphony-wdk/blob/822cacf4b162055880dd89fe1c1c8afb98b728e6/workflow-bot-app/src/test/java/com/symphony/bdk/workflow/BranchingIntegrationTest.java#L37
swadl: https://github.com/finos/symphony-wdk/blob/822cacf4b162055880dd89fe1c1c8afb98b728e6/workflow-bot-app/src/test/resources/branching/if-else-if.swadl.yaml



We would prefer to test with an unsatisfied condition (if: variables.foo != "foo") rather than (.. == "foo") to verify the condition works. (or both cases actually)

Reply:
👍  new test is removed, it is indeed kinda duplicated, so keep the existing test only.